buergerbuero/index.md

```diff
@@ -0,0 +1,137 @@
+# Bürgerbüro
+
+**Autor:** Finn Bechinka
+
+## Überblick
+
+Der Bürgerbüro Microservice soll Informationen und städtische Dienstleistungen zur Verfügung stellen.
+
+## Funktionale Anforderungen
+
+- Informationen bereitstellen
+  - Abfallkalender
+- Termine Vereinbaren
+  - An- und Ummelden
+  - Beantragung Ausweisdokumente
+  - Belaubiegungen
+  - Fischereischein
+  - Ausländer-Angelegenheiten
```

Comment:
Vielleicht etwas diskriminierend?

Reply:
das steht so auf der seite von meiner stadt 

das nennt nich wohl echt so 
hab grad gegooglet und das nennt sich bei anderen städten auch so

Reply:
Ok, dann is ok^^